docs/guides/Package-authoring-best-practices.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Package authoring best practices
+description: A general guide of best practices for creating high quality NuGet packages.
+author: chgill-MSFT
+ms.author: chgill
+ms.date: 09/17/2020
+ms.topic: conceptual
+---
+
+# Package authoring best practices
+
+This guidance is intended to give NuGet.org package authors a lightweight end to end reference for creating and publishing high quality packages. It will primarily focus on package-specific best practices such as metadata and packing. For more in-depth suggestions for building high quality libraries, see the .NET [Open-source library guidance](https://docs.microsoft.com/dotnet/standard/library-guidance/).
```

Comment:
What do you think of this?

```suggestion
This guidance is intended to give NuGet.org package authors a lightweight reference to create and publish high-quality packages. It will primarily focus on package-specific best practices such as metadata and packing. For more in-depth suggestions for building high quality libraries, see the .NET [Open-source library guidance](https://docs.microsoft.com/dotnet/standard/library-guidance/).
```